gravitee-apim-portal-webui-next/src/app/documentation/components/documentation-folder/documentation-folder.component.ts

```diff
@@ -40,7 +42,7 @@ export class DocumentationFolderComponent {
 
   pageIdEmitter$ = new Subject<string>();
   pageId = toSignal(this.pageIdEmitter$, { initialValue: null });
-  selectedPageContent = toSignal(this.pageIdEmitter$.pipe(switchMap(this.loadPageContent.bind(this))), { initialValue: '' });
+  selectedPageContent = toSignal(this.pageIdEmitter$.pipe(switchMap(this.loadPageContent.bind(this))), { initialValue: undefined });
```

Comment:
How about using `null` instead of `undefined` here? Null better represents an empty value whereas undefined can be ambiguous.

```suggestion
  selectedPageContent = toSignal(this.pageIdEmitter$.pipe(switchMap(this.loadPageContent.bind(this))), { initialValue: null });

Reply:
Done!